website/enroll/models.py

```diff
@@ -7,7 +7,7 @@ class VerifyCodeModel(models.Model):
     email = models.EmailField(max_length=36, unique=True)
     # this field allows at least 0-2147483647
     code = models.PositiveIntegerField()
-    #send_time = models.DurationField  # TODO
+    send_time = models.DateTimeField  # TODO
 
     # TODO: auto purge out of date's verify code
     # https://geek-docs.com/django/django-questions/80_django_can_stale_content_types_be_automatically_deleted_in_django.html#:~:text=%E5%9C%A8Django%E4%B8%AD%EF%BC%8C%E5%8F%AF%E4%BB%A5%E9%80%9A%E8%BF%87%E8%87%AA%E5%AE%9A%E4%B9%89%E7%AE%A1%E7%90%86%E5%91%BD%E4%BB%A4%E5%92%8C%E4%BF%A1%E5%8F%B7%E9%87%8F%20%28Signal%29%E6%9D%A5%E5%AE%9E%E7%8E%B0%E8%87%AA%E5%8A%A8%E5%88%A0%E9%99%A4%E8%BF%87%E6%9C%9F%E7%9A%84%E5%86%85%E5%AE%B9%E7%B1%BB%E5%9E%8B%E3%80%82%20%E9%A6%96%E5%85%88%EF%BC%8C%E6%88%91%E4%BB%AC%E5%8F%AF%E4%BB%A5%E5%88%9B%E5%BB%BA%E4%B8%80%E4%B8%AA%E8%87%AA%E5%AE%9A%E4%B9%89%E7%9A%84%E7%AE%A1%E7%90%86%E5%91%BD%E4%BB%A4%EF%BC%8C%E7%94%A8%E6%9D%A5%E6%A3%80%E6%B5%8B%E5%92%8C%E5%88%A0%E9%99%A4%E8%BF%87%E6%9C%9F%E7%9A%84%E5%86%85%E5%AE%B9%E7%B1%BB%E5%9E%8B%E3%80%82,%E5%9C%A8%E8%BF%99%E4%B8%AA%E5%91%BD%E4%BB%A4%E4%B8%AD%EF%BC%8C%E6%88%91%E4%BB%AC%E5%8F%AF%E4%BB%A5%E9%80%9A%E8%BF%87%20ContentType.objects.all%28%29%20%E6%96%B9%E6%B3%95%E8%8E%B7%E5%8F%96%E6%89%80%E6%9C%89%E7%9A%84%E5%86%85%E5%AE%B9%E7%B1%BB%E5%9E%8B%EF%BC%8C%E7%84%B6%E5%90%8E%E5%88%A4%E6%96%AD%E6%AF%8F%E4%B8%AA%E5%86%85%E5%AE%B9%E7%B1%BB%E5%9E%8B%E6%98%AF%E5%90%A6%E8%BF%87%E6%9C%9F%EF%BC%8C%E5%A6%82%E6%9E%9C%E8%BF%87%E6%9C%9F%E5%88%99%E5%88%A0%E9%99%A4%E3%80%82%20%E5%8F%AF%E4%BB%A5%E6%A0%B9%E6%8D%AE%E5%BA%94%E7%94%A8%E7%9A%84%E9%9C%80%E6%B1%82%E5%AE%9A%E4%B9%89%E8%BF%87%E6%9C%9F%E7%9A%84%E5%88%A4%E6%96%AD%E6%9D%A1%E4%BB%B6%EF%BC%8C%E6%AF%94%E5%A6%82%E6%A0%B9%E6%8D%AE%E4%BF%AE%E6%94%B9%E6%97%B6%E9%97%B4%E3%80%81%E6%9C%80%E5%90%8E%E8%AE%BF%E9%97%AE%E6%97%B6%E9%97%B4%E7%AD%89%E3%80%82
```

Comment:
After finished, just delete this TODO

Reply:
`TODO` is a label to remind this shall be done in the future. Once it's done, such a label is of no use any more.